playbook.yml

```diff
@@ -1,14 +1,26 @@
 ---
 - hosts: all
-  #remote_user: username specified in /etc/ansible/hosts
   become: yes
   become_user: root
+  gather_facts: False
+
+  pre_tasks:
+  - name: Install Python for Ansible
+    run_once: inventory_hostname == 'vagrant'
+    raw: |-
+      test -e /usr/bin/python || \
+        (apt -y update && sudo apt install -y python-minimal)
+    changed_when: False
+
+  - name: Install pip
+    run_once: inventory_hostname == 'vagrant'
+    apt:
+      name: python-pip
```

Comment:
Ansible works with python3. When I installed this on the aws box I used the following flag to tell it to use python3. Perhaps we should enable this everywhere and skip installing python2 and pip on vagrant. https://github.com/bocoup/web-platform-tests.live/blob/master/inventory/production#L5

Reply:
Yes, totally! Fixing this now